tests/Feature/Sensors/OutgoingRequestSensorTest.php

```diff
@@ -214,7 +214,10 @@ public function test_it_can_use_guzzle_directly(): void
             $stack->setHandler(new CurlHandler);
             $stack->push(Nightwatch::guzzleMiddleware());
             $client = new Client(['handler' => $stack]);
-            $client->get('https://laravel.com');
+            $client->get('https://laravel.com', [
+                'connection_timeout' => 1,
+                'timeout' => 1,
+            ]);
```

Comment:
This is testing guzzle and does a real request. Setting timeouts here so that it fails quickly.